README.md

```diff
@@ -5,8 +5,8 @@
 
 # Welcome to Superstaq!
 This repository is the home of Superstaq development team's open source work, which includes:
-* [Superstaq](https://www.infleqtion.com/superstaq), our quantum software platform that is optimized across the quantum stack and enables users to write quantum programs in Cirq or Qiskit and target a variety of quantum computers and simulators
-* [Supermarq](https://www.infleqtion.com/supermarq), our quantum benchmarking suite
+* Our quantum software platform that is optimized across the quantum stack and enables users to write quantum programs in Cirq or Qiskit and target a variety of quantum computers and simulators. Read more about it [here](https://www.infleqtion.com/superstaq).

```

Comment:
This could just be a preference thing, but should we have the names 'Superstaq' and 'Supermarq' like previously -- or were they removed due to redundancy/repetition? Not really a big deal either way.